python/cuml/manifold/umap.pyx

```diff
@@ -871,7 +881,7 @@ class UMAP(Base,
             "a",
             "b",
             "target_n_neighbors",
-            "target_weights",
+            "target_weight",
```

Comment:
We might need to do an exception in `test_base_children_get_param_names` for the renamed parameter during 21.06, could you also create an issue to remove the parameter from 21.08 and the test exception?